server/src/bin/main.rs

```diff
@@ -20,6 +21,12 @@ async fn main() {
 
     let (msg_bus_tx, mut msg_bus_rx) = broadcast::channel::<Msg>(MSG_BUS_QUEUE_SIZE);
     let quote_book = InMemoryQuoteBook::default();
+    // Open the ledger db
```

Comment:
Nit: I like empty new lines between logical blocks.
```suggestion

    // Open the ledger db
```